src/acl/Acl.h

```diff
@@ -61,9 +61,11 @@ class ACL
 
     /// \returns (linked) Options supported by this ACL
     virtual const Acl::Options &options() { return Acl::NoOptions(); }
+    /// \returns (linked) 'line' Options supported by this ACL
+    const Acl::Options &lineOptions();
```

Comment:
When viewed in isolation, options() imply that it returns both line and global options (because the description says nothing about global or line options).

When viewed together, these declarations imply that options() do not contain lineOptions() (because the former collection is empty (by default) even when/if the latter collection is not).

We should fix and/or re-describe options() accordingly.